### ISSUE TITLE
Bug:  godoc uses len(s) for max_len, resulting wrong calculations for utf8 strings

### ISSUE THREAD

**Issue by ravsii** [godoc-lint/godoc-lint#58]
See this code for example:
```
// Limit - количество запрашиваемых элементов для запроса.
func (r *RequestAgg) Limit() int {
	return r.limit
}
```

<img width="1356" height="129" alt="Image" src="https://github.com/user-attachments/assets/0e06b382-7946-4a24-8839-2add5d953661" />

using
```
max-len:
  length: 77
```

Expected max_len: 56 (53 without `// `), and no errors, see https://go.dev/play/p/gvFGHQH0otN

I believe the problem is line 84
https://github.com/godoc-lint/godoc-lint/blob/2636f409fbda85cb5abf621e24f16469cfe57d7d/pkg/check/max_len/max_len.go#L83-L90

Would love to PR, the fix seems small enough

**Comment by babakks** [comment]
Good catch, @ravsii! Thanks for reporting. I really appreciate if you can make a PR for this! 🍻
